src/main/scala/mesosphere/marathon/core/group/impl/GroupManagerImpl.scala

```diff
@@ -129,13 +129,18 @@ class GroupManagerImpl(
           case Left(left) =>
             Left(left)
           case Right(changed) =>
+            // Set default roles
```

Comment:
Is this a todo? Or should it be part of #6959?

Reply:
It's a leftover from the previous approach. Removed it.